demo/front.js

```diff
@@ -1,35 +1,55 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
 import Logo from 'svg-react-loader?name=Logo!./frontside-icon-transparent.svg';
-import Paneset from '@folio/stripes-components/lib/Paneset';
-import Pane from '@folio/stripes-components/lib/Pane';
 import style from './front.css';
 
 export default function Front() {
   return (
     <div className={style.front}>
-      <h1><Logo className={style.logo}/> Frontside FOLIO</h1>
+      <h1><Logo className={style.logo}/> e-Holdings, by Frontside</h1>
       <p>
         Welcome to the Frontside FOLIO demo. Here you'll find a showcase of the applications and modules that we've been working on.
       </p>
 
-      <p>While we wire up the general search and navigation system for e-holdings, please have a look at these direct links as a starting point:</p>
+      <p>
+        <em>HTTPS is currently only partially enabled at the moment for this demo. To allow insecure content to load from the demo Okapi instance, you'll need to temporarily allow "unsafe scripts" in your browser:</em>
+      </p>
+      <ul>
+        <li>
+          <a href="https://superuser.com/questions/487748/how-to-allow-chrome-browser-to-load-insecure-content">Chrome instructions</a>
+        </li>
+      </ul>
+
+      <p>
+        While we wire up the general search and navigation system for e-holdings, please have a look at these direct links as a starting point:
+      </p>
+
+      <h3>Search</h3>
+      <ul>
+        <li><Link to="/eholdings/vendors">Search for a vendor</Link></li>
```

Comment:
i wondered this too, but once you visit the vendor search it becomes pretty apparent how to navigate from there.